src/quickstart/provider/azure-acs/install-azure-acs.md

```diff
@@ -0,0 +1,75 @@
+# Install Deis Workflow on Azure Container Service
+
+## Check Your Setup
+
+First check that the `helm` command is available and the version is v2.0.0 or newer.
+
+```
+$ helm version
+Client: &version.Version{SemVer:"v2.0.0", GitCommit:"51bdad42756dfaf3234f53ef3d3cb6bcd94144c2", GitTreeState:"clean"}
+Server: &version.Version{SemVer:"v2.0.0", GitCommit:"51bdad42756dfaf3234f53ef3d3cb6bcd94144c2", GitTreeState:"clean"}
+```
+
+Finally, intialize Helm:
+```
+helm init
+```
+
+Ensure the `kubectl` client is installed and can connect to your Kubernetes cluster.
+
+## Add the Deis Chart Repository
+
+The Deis Chart Repository contains everything needed to install Deis Workflow onto a Kubernetes cluster, with a single `helm install deis/workflow --namespace deis` command.
+
+Add this repository to Helm:
+
+```
+$ helm repo add deis https://charts.deis.com/workflow
+```
+
+## Install Deis Workflow
+
+Now that Helm is installed and the repository has been added, install Workflow by running:
+
+```
+$ helm install deis/workflow --namespace deis --set controller.docker_tag=v2.9.0-acs,controller.org=kmala
```

Comment:
Is this still relevant today? Do we want to ship this doc before v2.10 is released or do we want to give ACS  some more time to fix it on their end first?